checks/evaluation/code_review.go

```diff
@@ -109,5 +105,13 @@ func reviewScoreForChangeset(changeset *checker.Changeset) (score reviewScore) {
 		}
 	}
 
+	dl.Info(
+		&checker.LogMessage{
+			Text: fmt.Sprintf(
+				"couldn't find approvals for revision: %s platform: %s",
```

Comment:
note: this displays the following for commits unassociated with a PR/MR.
Should we have a value for when no review platform detected?
```
        "Info: couldn't find approvals for revision: cf79bed7ddc85c762a291400f904d18fbbf14581 platform: ",
        "Info: couldn't find approvals for revision: aca3970538e89a86be9968052615ae5ed05d9259 platform: ",
        "Info: couldn't find approvals for revision: 8496c8c8f2f9027d4ba12ce1e5f7a79b52fd29e9 platform: ",
```

Reply:
Added `committed directly`